tests/integration/data/awardFinancialValid.csv

```diff
@@ -1,9 +1,9 @@
+19,72,,,X,306,0,,ABCD,123456807,12-34-56-78-90-AS-DF-AB-XZ-YW,Overseas Programs,2,1255,D,6500,6500,13000,6500,6500,6500,6500,6500,13000,6500,6500,6500,6500,13000,32500,6500,13000,6500,6500,6500,6500,6500,19500,6500,6500,6500,6500,6500,6500,6500,6500,6500,26000,Q
+,19,2016,2016,,113,0,,ABCD,123456799,12-34-56-78-90-AS-DF-AB-XZ-YW, , ,1255,D,6500,6500,13000,6500,6500,6500,6500,6500,13000,6500,6500,6500,6500,13000,32500,6500,13000,6500,6500,6500,6500,6500,19500,6500,6500,6500,6500,6500,6500,6500,6500,6500,26000,Q
+,28,,,X,406,0,,ABCD,123456800,12-34-56-78-90-AS-DF-AB-XZ-YW,Overseas Programs,2,1255,D,6500,6500,13000,6500,6500,6500,6500,6500,13000,6500,6500,6500,6500,13000,32500,6500,13000,6500,6500,6500,6500,6500,19500,6500,6500,6500,6500,6500,6500,6500,6500,6500,26000,Q
+,28,2010,2011,,406,0,,ABCD,123456801,12-34-56-78-90-AS-DF-AB-XZ-YW,Overseas Programs,2,1255,D,6500,6500,13000,6500,6500,6500,6500,6500,13000,6500,6500,6500,6500,13000,32500,6500,13000,6500,6500,6500,6500,6500,19500,6500,6500,6500,6500,6500,6500,6500,6500,6500,26000,Q
+69,13,,,X,2050,5,,ABCD,123456802,12-34-56-78-90-AS-DF-AB-XZ-YW,Overseas Programs,2,1255,D,6500,6500,13000,6500,6500,6500,6500,6500,13000,6500,6500,6500,6500,13000,32500,6500,13000,6500,6500,6500,6500,6500,19500,6500,6500,6500,6500,6500,6500,6500,6500,6500,26000,Q
 28,28,,,X,8007,0,ABCD,ABCD,,,Overseas Programs,2,1255,D,6500,6500,13000,6500,6500,6500,6500,6500,13000,6500,6500,6500,6500,13000,32500,6500,13000,6500,6500,6500,6500,6500,19500,6500,6500,6500,6500,6500,6500,6500,6500,6500,26000
 ,49,,,X,100,0,ABCD,ABCD,,,Overseas Programs,2,1255,D,6500,6500,13000,6500,6500,6500,6500,6500,13000,6500,6500,6500,6500,13000,32500,6500,13000,6500,6500,6500,6500,6500,19500,6500,6500,6500,6500,6500,6500,6500,6500,6500,26000
 ,49,2010,2011,,100,0,ABCD,ABCD,,,Overseas Programs,2,1255,D,6500,6500,13000,6500,6500,6500,6500,6500,13000,6500,6500,6500,6500,13000,32500,6500,13000,6500,6500,6500,6500,6500,19500,6500,6500,6500,6500,6500,6500,6500,6500,6500,26000

```

Comment:
Any reason why these are blank whereas the others have 'Q'?

Reply:
I forgot and we clearly don't use this file because it didn't throw an error when half the rows didn't have the right number of columns, that's all